lavalink/WebSocket.py

```diff
@@ -56,42 +58,71 @@ def __init__(self, lavalink, host, password, ws_port, ws_retry, shard_count):
                 for task in self._queue:
                     await self.send(**task)
 
+    async def _attempt_reconnect(self) -> bool:
+        """
+        Attempts to reconnect to the lavalink server.
+
+        Returns
+        -------
+        bool
+            ``True`` if the reconnection attempt was successful.
+        """
+        await self._ws.close()

```

Comment:
`connect` does checks for open websockets beforehand so you could probably omit this line